src/operator/main.go

```diff
@@ -221,6 +226,10 @@ func main() {
 		logrus.WithError(err).Fatal("unable to set up ready check")
 	}
 
+	if !enforcementEnabledGlobally {
+		logrus.Info("Running with %s=false, won't perform any enforcement", enableEnforcementKey)
```

Comment:
@amit7itz the linter is right :) use Infof

Reply:
nice 😬